packages/package-values.yaml

```diff
@@ -36,10 +36,10 @@ managementPackageRepository:
       version: latest
       #! Gets replaced with imgpkg sha256 at build, this should be name:version
       sha256: "core-management-plugins:latest"
-    - name: addons-manager
-      displayName: addons-manager
+    - name: addons-manager-v2
+      displayName: addons-manager-v2
       #! Relative path to package bundle
-      path: packages/management/addons-manager
+      path: packages/management/addons-manager-v2
```

Comment:
i think just changing the name on lines 39 and 40 is sufficient and renaming the package directory name is not needed

Reply:
@yharish991 yeah I have the same feeling, let me update it.

Reply:
@yharish991 Hi Harish, I think this directory has to be change because bolt will looking for this name in directory for packaging